src/Leanplum.js

```diff
@@ -136,6 +136,30 @@ export default class Leanplum {
     InternalState.removeVariablesChangedHandler(handler)
   }
 
+  static forceContentUpdate(callback) {
+    LeanplumRequest.request(Constants.METHODS.GET_VARS,
+      new ArgsBuilder()
+      .add(Constants.PARAMS.INCLUDE_DEFAULTS, false), {
+        queued: false,
+        sendNow: true,
+        response: function (response) {
+          let getVarsResponse = LeanplumRequest.getLastResponse(response);
+          let success = false;
```

Comment:
We can use isResponseSuccess() here. That way we don't have to explicitly set it 'true' later.
Also it is easier to read ;).